src/Service.Host/client/src/components/StockMove.js

```diff
@@ -177,16 +177,16 @@ function StockMove({
         }
     };
 
-    const handleSelectRow = row => {
-        setSelectedRow(availableStock[row.rowIds[0]]);
+    const handleSelectRow = rows => {
+        setSelectedRow(availableStock[rows[0]]);
```

Comment:
So does it now just give you a list of rowIds on select?

Reply:
yep and doesn't stringify them anymore